### DISCUSSION TITLE
OAuth2 user clarification

### DISCUSSION THREAD

**Discussion by enifsieus** [pretix/pretix#2535] [Q&A] [answered]
After setting up an OAuth2 connection, it seems that this is only possible to connect using staff logins/profiles (i.e. pretixbase_user), and not with pretixbase_customer logins. Is there any way to federate Customer login and access Customer profiles? Assuming not, the documentation might be clarified a bit in this regard.  Thanks!

**Comment by raphaelm** [comment, marked as answer]
No, currently not. OAuth2 capabilities currently only refer to "users", which are backend users.

We do plan on OAuth2 support for customer accounts. Hopefully this year, but I can't promise it at this time.
